cli/scripts/webos/run.js

```diff
@@ -1,6 +1,7 @@
 const path = require('path');
 const fs = require('fs');
 const chalk = require('chalk');
+const exec = require('child_process').exec;
```

Comment:
What do you think by replace this line and next one by:
```js
cons { exec, execSync } = require('child_process');
```

Reply:
Nice!!